packages/react/src/components/media/players/AudioPlayer.tsx

```diff
@@ -13,40 +13,58 @@ const mediaControllerSelector = ({
   fullscreen,
 });
 
-export type { AudioPlayerProps };
-
-export const AudioPlayer = React.forwardRef<
+export type AudioPlayerProps = AudioPlayerCoreProps<
   HTMLAudioElement,
-  AudioPlayerProps<HTMLAudioElement, PosterSource>
->(({ src, autoPlay, title, loop, muted, objectFit }, ref) => {
-  const { fullscreen } = useMediaController(mediaControllerSelector);
+  PosterSource
+> & {
+  allowCrossOriginCredentials?: boolean;
+};
 
-  const filteredSources = React.useMemo(() => {
-    return src?.filter((s) => s?.mime && canPlayMediaNatively(s)) ?? [];
-  }, [src]);
+export const AudioPlayer = React.forwardRef<HTMLAudioElement, AudioPlayerProps>(
+  (
+    {
+      src,
+      autoPlay,
+      title,
+      loop,
+      muted,
+      objectFit,
+      allowCrossOriginCredentials,
+    },
+    ref,
+  ) => {
+    const { fullscreen } = useMediaController(mediaControllerSelector);
 
-  return (
-    <audio
-      className={styling.media.audio({
-        size: fullscreen ? 'fullscreen' : objectFit,
-      })}
-      loop={loop}
-      aria-label={title ?? 'Audio player'}
-      role="audio"
-      autoPlay={autoPlay}
-      ref={ref}
-      webkit-playsinline="true"
-      playsInline
-      muted={muted}
-    >
-      {filteredSources.map((source) => (
-        <source key={source.src} src={source.src} type={source.mime!} />
-      ))}
-      {
-        "Your browser doesn't support the HTML5 <code>audio</code> tag, or the audio format."
-      }
-    </audio>
-  );
-});
+    const filteredSources = React.useMemo(() => {
+      return src?.filter((s) => s?.mime && canPlayMediaNatively(s)) ?? [];
+    }, [src]);
+
+    return (
+      <audio
+        className={styling.media.audio({
+          size: fullscreen ? 'fullscreen' : objectFit,
+        })}
+        loop={loop}
+        aria-label={title ?? 'Audio player'}
+        role="audio"
+        autoPlay={autoPlay}
+        ref={ref}
+        webkit-playsinline="true"
+        playsInline
+        muted={muted}
+        crossOrigin={
+          allowCrossOriginCredentials ? 'use-credentials' : 'anonymous'
```

Comment:
Is anonymous the default? Either way, what happens if we set it as `undefined` instead to leave it for the browser to decide the default?